package.json

```diff
@@ -78,7 +78,7 @@
     "test-wpt": "mocha test/web-platform-tests/index.js",
     "test-tuwpt": "mocha test/web-platform-tests/to-upstream.js",
     "test-mocha": "mocha",
-    "test-mocha-all": "mocha test/index.js",
+    "test-mocha-all": "mocha --bail test/index.js",
```

Comment:
I'd rather not make this change, as especially on CI it's important to see all failures